README.md

```diff
@@ -59,12 +59,18 @@ $ mvn -Declipse.target=neon package
 
 ### Configuring Maven/Tycho Toolchains
 
+_Note: this section is only relevant for configuring CI builds_
```

Comment:
I'd add this to the section title like `Configuring Maven/Tycho Toolchains in CI builds`